components/p2p/component.go

```diff
@@ -11,16 +11,20 @@ import (
 	"github.com/libp2p/go-libp2p/core/host"
 	"go.uber.org/dig"
 
+	ma "github.com/multiformats/go-multiaddr"
```

Comment:
why the custom import label?

Reply:
This is how it is imported in the libp2p literature and example code.

Reply:
Changed